db/migrate/20200525073201_drop_puppet_class_id_from_look_up_key.rb

```diff
@@ -0,0 +1,5 @@
+class DropPuppetClassIdFromLookUpKey < ActiveRecord::Migration[6.0]
+  def up
+    remove_column :lookup_keys, :puppetclass_id
+  end
```

Comment:
This also needs a `down`, or use `change` since rails knows how to reverse remove_column

Reply:
@tbrisker done :)

Reply:
i think you forgot to git add :)

Reply:
@tbrisker  Ops! should be ok now